modules/ingest-geoip/src/main/java/org/elasticsearch/ingest/geoip/DatabaseRegistry.java

```diff
@@ -308,12 +308,12 @@ void retrieveAndUpdateDatabase(String databaseName, GeoIpTaskState.Metadata meta
 
     void updateDatabase(String databaseFileName, String recordedMd5, Path file) {
         try {
-            LOGGER.info("database file changed [{}], reload database...", file);
             DatabaseReaderLazyLoader loader = new DatabaseReaderLazyLoader(cache, file, recordedMd5);
             DatabaseReaderLazyLoader existing = databases.put(databaseFileName, loader);
             if (existing != null) {
                 existing.close();
             }
+            LOGGER.info("reloaded changed database file [{}]", file);
```

Comment:
Rewording suggestion: worth mentioning in the message that we're talking about the geoip database (folk don't read the logger name much) and also adding `successfully` just to add to the overall happiness of the message :grin:

```suggestion
            LOGGER.info("successfully reloaded changed geoip database file [{}]", file);
```